lib/matplotlib/_mathtext_data.py

```diff
@@ -910,7 +910,7 @@
     'O'                        : 216,
     'hookleftarrow'            : 8617,
     'trianglerighteq'          : 8885,
-    'nsime'                    : 8772,
```

Comment:
Has this just been wrong from 2006 ?!

Reply:
I guess so. However, I do not think it was documented so I wonder if someone ever used it?